client/extensions/woocommerce/state/action-types.js

```diff
@@ -14,6 +14,9 @@ export const WOOCOMMERCE_PRODUCT_CREATE_SUCCESS = 'WOOCOMMERCE_PRODUCT_CREATE_SU
 export const WOOCOMMERCE_PRODUCT_EDIT = 'WOOCOMMERCE_PRODUCT_EDIT';
 export const WOOCOMMERCE_PRODUCT_UPDATED = 'WOOCOMMERCE_PRODUCT_UPDATED';
 export const WOOCOMMERCE_PRODUCT_VARIATION_EDIT = 'WOOCOMMERCE_PRODUCT_VARIATION_EDIT';
+export const WOOCOMMERCE_REQUEST = 'WOOCOMMERCE_REQUEST';
```

Comment:
Please consider using JETPACK_WOOCOMMERCE_REQUEST, JETPACK_WOOCOMMERCE_REQUEST_SUCCESS etc instead to disambiguate from WOOCOMMERCE requests that terminate at WPCOM